tools/icu/current_ver.dep

```diff
@@ -1,14 +1,6 @@
 [
   {
-    "url": "https://github.com/unicode-org/icu/releases/download/release-63-1/icu4c-63_1-src.zip",
-    "md5": "d25bc38089db64668fd86f34b4aa9b93"
-  },
-  {
-    "url": "https://sourceforge.net/projects/icu/files/ICU4C/63.1/icu4c-63_1-src.zip",
-    "md5": "d25bc38089db64668fd86f34b4aa9b93"
-  },
-  {
-    "url": "https://download.icu-project.org/files/icu4c/63.1/icu4c-63_1-src.zip",
-    "md5": "d25bc38089db64668fd86f34b4aa9b93"
+    "url": "https://github.com/unicode-org/icu/releases/download/release-64-2/icu4c-64_2-src.tgz",
```

Comment:
could add `http://download.icu-project.org/files/icu4c/64.2/icu4c-64_2-src.tgz` (not https!!) as a second entry with the same version # and hash as a backup.

Reply:
@srl295 sorry for removing it, I was unable to find this link on their website. I'll add this as you prefer. Could you tell me where I could have found the link?

Reply:
😇